server/src/main/java/org/eclipse/lsp/cobol/service/CobolTextDocumentService.java

```diff
@@ -454,4 +462,18 @@ private String createDescriptiveErrorMessage(String action, String uri) {
   private BiConsumer<Object, Throwable> reportExceptionIfThrown(String message) {
     return (res, ex) -> ofNullable(ex).ifPresent(it -> LOG.error(message, it));
   }
+
+  private List<FoldingRange> getFoldingRangeFromDocumentSymbol(
+      List<DocumentSymbol> documentSymbols) {
+    List<FoldingRange> foldingRanges = new ArrayList<>();
+    if (documentSymbols.isEmpty()) foldingRanges.addAll(Collections.emptyList());
```

Comment:
Nice trick! But the line does nothing =)
Can you delete it?

Reply:
Thanks
I will update